### PR TITLE
feat: get organization users

### PR DESCRIPTION
**Pull request by chukssomzzy**
## Description
This feature allows getting all users in an organization. It facilitates the retrieving of users in an organization by pagination of the users for effective retrieval

## Related Issue
[FEAT: Fetch all Users of an Organisation API - BACKEND](https://github.com/hngprojects/hng_boilerplate_python_fastapi_web/issues/78)

## Motivation and Context
This change is necessary to provide functionality for getting all organization users, enhancing the flexibility and usability of the application. It solves the problem of not being able to get all organization users.

## How Has This Been Tested?
- [X] Tested extensively with Postman.
- [X] Written working unittests with pytest.

## Screenshots (if appropriate):
image

## Types of changes
- [X] Bug fix (non-breaking change which fixes an issue).
- [X] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist:
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: api/v1/routes/organization.py
Comment (joboy-dev) [inline api/v1/routes/organization.py:1]: Do this
Pull latest changes and use the paginated_response function in api/utils/pagination.py to generate a paginated response
Thanks

**Review by joboy-dev — CHANGES_REQUESTED** [review]
Make changes

File: api/v1/routes/organization.py
Comment (chukssomzzy) [reply api/v1/routes/organization.py:1]: It not possible to use paginated_response function, since it doesn't do a join.

**Comment by chukssomzzy** [comment]
> Make changes

requested changes has been made. 
I am using paginate_response to paginate response and I have merged current changes

File: api/v1/routes/organization.py
Comment (joboy-dev) [reply api/v1/routes/organization.py:1]: Oh. So you were doing a join?
Okat

**Review by joboy-dev — APPROVED** [review]

**Review by joboy-dev — APPROVED** [review]

**Review by joboy-dev — APPROVED** [review]

**Review by CodewithSegNet — APPROVED** [review]

**Review by joboy-dev — APPROVED** [review]

**Pull request merged**